### PR TITLE
{devel}[foss/2021a] f90wrap v0.2.8 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/f/f90wrap/f90wrap-0.2.8-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/f2py-f90wrap', 'bin/f90doc', 'bin/f90wrap'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/f/f90wrap/f90wrap-0.2.8-foss-2021a.eb:31]: @lexming can you add a `sanity_check_commands` here too?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/a82b455ed81b1ba2fa977c0c2234671f for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**